plugins/plugin-react-refresh/plugin.js

```diff
@@ -45,6 +45,7 @@ async function transformJs(contents, id, cwd, skipTransform) {
     );
     fastRefreshEnhancedCode = contents;
   } else {
+    const {plugins = [], ...restConfig} = babelConfig instanceof Object ? babelConfig : {};
```

Comment:
I think we should test if someone specifies `{ babel: true }`. I believe it‘ll throw at `...restConfig` below.

What do you think about simply throwing an error if `babel` is strictly equal to `true`? For example: `Error: option "babel" either be 'false' or a valid Babel config. Received 'true'`.

Reply:
I know it’s not a valid option, but it would be helpful to users if a friendly error was shown.

Reply:
We may also want to check for `string`s, as well, in case someone attempts to provide a path to their Babel config (IMO just throw in that case too; if we want to add importing a config file later we can)